python/SWESimulators/Common.py

```diff
@@ -23,9 +25,9 @@ def new_func(*args, **kwargs):
         return func(*args, **kwargs)
     return new_func
 
-def get_kernel(cl_ctx, kernel_filename, block_width, block_height):
+def get_kernel(kernel_filename, block_width, block_height):
     """
-    Static function which reads a text file and creates an OpenCL kernel from that
+    Static function which reads a text file and creates an CUDA kernel from that
     """
     #Create define string
     define_string = "#define block_width " + str(block_width) + "\n"
```

Comment:
Can this be moved into a compile-time template type thing now?

Reply:
Yes. We should do that in a separate PR/commit.